Chumigos/Chumigos/Games/Avatar Game/View/AvatarGameView.swift

```diff
@@ -136,16 +130,16 @@ struct AvatarGameView: View {
                                     .multilineTextAlignment(.center)
                                 
                                 Grid<AvatarGameTile>(rows: numberOfRows, columns: numberOfColumns, spacing: screenWidth * 0.008) { (row, column) in
-                                    AvatarGameTile(facePart: self.viewModel.roundFaceParts[(row * self.numberOfColumns)+column], eyeImage: self.$viewModel.eyeImage, mouthImage: self.$viewModel.mouthImage, eyebrowImage: self.$viewModel.eyebrowImage, confirmPressed: self.$buttonIsPressed)
+                                    AvatarGameTile(facePart: self.viewModel.roundFaceParts[(row * self.numberOfColumns)+column], eyeImage: self.$viewModel.eyeImage, mouthImage: self.$viewModel.mouthImage, eyebrowImage: self.$viewModel.eyebrowImage, confirmPressed: self.$viewModel.confirmPressed)
```

Comment:
Next time add a comment telling us what is this math equation to get matrix position as an array